caravel/templates/caravel/base.html

```diff
@@ -8,10 +8,10 @@
 
   {% block head_js %}
     {{super()}}
-    <script src="/static/assets/dist/css-theme.entry.js"></script>
+    <script src="/static/assets/dist/css-theme.{{VERSION_STRING}}.entry.js"></script>
```

Comment:
NIT: we usually leave room with spaces between brackets and variables in jinja tags as in:
`{{ FOO }}`